### PR TITLE
[11.0] [MIG] account_payment_mandate

### PR DESCRIPTION
**Pull request by etobella**
Migration to 11.0.
Some tests were added
Depends on:
 - [x] account_payment_mode #399
 - [x] account_payment_partner #400
 - [x] account_payment_order #401

### CODE REVIEW

**Comment by pedrobaeza** [comment]
One thing pending in the roadmap is to add in this generic module a field with type selector that includes here a "Generic mandate", and account_sepa_direct_debit simply adds a value instead of overwriting all fields and menus from this generic one.

File: account_banking_mandate/README.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Configuration
+=============
+
+TODO
+
+Usage
+=====
+
+To use this module, see menu "Accounting > payment > SEPA direct debit mandates"
```
Comment (rgarnau) [inline account_banking_mandate/README.rst]: This does not match with the views. I think the menu would be 'Invoicing > Payments > Banking Mandates' instead.

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review.

**Review by HviorForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by angelmoya** [comment]
Functional tested OK :+1:

**Pull request merged**